tasks/task646_dailydialog_classification.json

```diff
@@ -0,0 +1,1238 @@
+{
+    "Contributors": [
+        "Karan Niraj Naik"
+    ],
+    "Source": [
+        "DailyDialog (http://yanran.li/dailydialog)"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "In this task, we are given a Dialy Dialog Dataset, in which each conversation has multiple dialogues and each dialogues in the conversation is annotated based on emotion information, dialogue act as well as topic of converstaion. Here we classify the data based on the topic of conversation. If the topic of conversation is Tourism in the one of the dialogues of the conversation then output Yes otherwise if the topic of conversation is not Tourism then output No",
```

Comment:
I think you can simplify the definition. I think you need only the last part where you say

```
Classify the given dialogue is about "Tourism". If the topic of conversation is Tourism in then output "Yes"; otherwise if the topic of conversation is not Tourism then output "No".
```
Similarly, you can simplify the rest of the tasks.